frontend/src/css/modules/analytics.scss

```diff
@@ -0,0 +1,38 @@
+li.analytics-list {
```

Comment:
Thanks for this. Giving a class to ul and doing the nesting can be a better approach ?